hack/common.sh

```diff
@@ -18,7 +19,7 @@ get_operator_sdk() {
 
   DOWNLOAD_DIR=/tmp/operator-sdk
   # TODO: Make this download the same version we have in go dependencies in gomod
-  wget --no-verbose -O $DOWNLOAD_DIR https://github.com/operator-framework/operator-sdk/releases/download/v0.19.4/operator-sdk-v0.19.4-x86_64-linux-gnu && chmod +x /tmp/operator-sdk || return
+  wget --no-verbose -O $DOWNLOAD_DIR https://github.com/operator-framework/operator-sdk/releases/download/v1.6.1/operator-sdk_linux_amd64 -o operator-sdk && chmod +x /tmp/operator-sdk || return
```

Comment:
You should call out in the commit that this is where you are actually swapping SDK versions.